INSTALL.md

```diff
@@ -558,28 +558,27 @@ This is a problem if you are running a keyboard driver for some non-english
 languages. These drivers catch the ^ character to produce the French circumflex
 accent and do not pass it properly to GAP. For macOS users, as a workaround
 please refer to the section "GAP for macOS" below for information on
-how to install readline and section 5 on how to recompile GAP, for windows no
+how to install readline and section 5 on how to recompile GAP, for Windows no
 fix is known. (One can type `POW(a,b)` for `a^b`.)
 
 ## Problems specific to Windows
 
-### Cut and Paste does not work
+The best way to use GAP in Windows is to use the "Windows Subsystem for Linux"
+( https://learn.microsoft.com/en-us/windows/wsl/install ), also known as WSL.
+This can be found in the "Microsoft Store" in Windows, and allows you to install
+a full copy of Ubuntu. You can then follow the guidance for building and using
```

Comment:
```suggestion
a full copy of Linux. You can then follow the guidance for building and using
```

Reply:
I mean ubuntu here, as that's actually what you have to go look for (Ubuntu), and Ubuntu is by far the best supported OS in WSL (I've tried others and they are always fragile).

Reply:
Then maybe explain it as such? The text as written is ok for someone who already knows what Linux and Ubuntu are, but my experience is that an average Windows user does not understand how these things relate...